support-frontend/assets/pages/new-contributions-landing/components/ContributionThankYou/ContributionThankYouBlurb.jsx

```diff
@@ -0,0 +1,21 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+// ----- Render ----- //
+
+function ContributionThankYouBlurb() {
+  return (
+    <div className="gu-content__blurb gu-content__blurb--thank-you">
+      <h1 className="gu-content__blurb-header">Thank you for <br className="gu-content__blurb-header-break" />a valuable <br className="gu-content__blurb-header-break" />contribution</h1>
```

Comment:
i'd try using `<wbr>` rather than changing this at a breakpoint using css, which is a bit hard to maintain

Reply:
I'll add to TBD